src/routes/docs/products/functions/executions/+page.markdoc

```diff
@@ -17,25 +17,25 @@ Here's the information shown on this table.
 - Column
 - Description
 ---
-- EXECUTION ID
+- `execution id`
 - Unique identifier for each execution
 ---
-- STATUS
+- `status`
 - The current status of the execution
 ---
-- CREATED
+- `created`
 - Timestamp of when the execution was created
 ---
-- TRIGGER
+- `trigger`
 - The [platform event](/docs/advanced/platform/events) that triggered the execution
 ---
-- METHOD
+- `method`
 - The HTTP method used to create the execution
 ---
-- PATH
+- `path`
 - The URL path the function execution was called with
 ---
-- DURATION
+- `duration`
 - The time taken for the execution
```

Comment:
@Meldiron, I don't think we should change these to be code formatted because these are column headings in a table in the Console and aren't code.

![image](https://github.com/user-attachments/assets/24c1bf06-3e5b-4ab0-bf1f-7408703e2b26)

If anything, they should be sentence case instead of upper case to match [Messages](https://appwrite.io/docs/products/messaging/messages#messages) and [Audit logs](https://appwrite.io/docs/advanced/security/audit-logs)

Reply:
I agree it can be sentence case